app/views/groups/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<html>
+  <head>
+     <%= stylesheet_link_tag "group_index", "data-turbo-track": "reload" %>
+  </head>
+
+  <body>
+    <p style="color: green"><%= notice %></p>
+      <div class="group-wrapper">
+      <div class="group-header">
+        <h2>Categories</h2>
+      </div>
+
+      <div id="groups">
+        <% @groups.each do |group| %>
+          <%= render group %>
+        <% end %>
+      </div>
+        <a href=<%= new_group_path %> class="wrapper-btn-new-group">
+          <button class="btn-new-group">
+            New Categorie
+          </button>
```

Comment:
- [OPTIONAL] Kindly follow the provided design (width, font, size, height, background, color...) Place the "NEW CATEGORY" button at the bottom of your page. ⬇️ 
 The transaction button has a similar problem. Kindly fix them as well. 💯 👍🏻 
- Original
![image](https://user-images.githubusercontent.com/79058364/232208901-d1037753-9d20-4bc3-9e9b-c239e1888386.png)